RollingStockOwnership/StartingConditions.cs

```diff
@@ -238,7 +248,7 @@ private static void EnsurePlayerHasRequiredItemsForSteamLoco()
 		List<ShopItemData> shopItemsData = GlobalShopController.Instance.shopItemsData;
 		Main.LogDebug(() => $"All shop items:\n{{\n\t{string.Join(",\n\t", shopItemsData.Select(itemData => itemData.item.name))}\n}}");
 
-		string [] itemNames = { "shovel", "lighter" };
+		string [] itemNames = { "shovel", "lighter","oiler" }; // adding all necessary items to the player inventory
```

Comment:
nit: Missing a space after the comma. I also don't think the comment is necessary as it's already explained by the method name, but feel free to leave it if you feel strongly about it.